include/mqtt.h

```diff
@@ -170,7 +170,8 @@ int __cheri_compartment("MQTT") mqtt_publish(Timeout    *t,
                                              const char *topic,
                                              size_t      topicLength,
                                              const void *payload,
-                                             size_t      payloadLength);
+                                             size_t      payloadLength,
+                                             bool        retain = false);
```

Comment:
Would it be possible to update the documentation of `mqtt_publish` to mention this new `retain` parameter?